src/rust/engine/Cargo.toml

```diff
@@ -19,15 +19,12 @@ cc = "1.0"
 # [workspace]
 
 [dependencies]
-blake2-rfc = "0.2.17"
+boxfuture = { path = "boxfuture" }
 fnv = "1.0.5"
+fs = { path = "fs" }
 futures = "0.1.16"
 futures-cpupool = "0.1.6"
```

Comment:
engine no longer uses futures-cpupool.

How long this will remain the case, I don't know, but might as well remove it for now.